beacon-chain/core/helpers/committee.go

```diff
@@ -307,42 +307,33 @@ func AttestationParticipants(
 	return participants, nil
 }
 
-// VerifyBitfield validates a bitfield with a given committee size.
+// VerifyBitfield Verify ``bitfield`` against the ``committee_size``.
```

Comment:
don't need `` for comments

```suggestion
// VerifyBitfield verifies bitfield against the committee size.
```

Reply:
thanks